### PR TITLE
Add Styling example docs

### PR DESCRIPTION
**Pull request by wselwood**
Add an example notebook of styling plots.

### CODE REVIEW

File: docs/source/examples/styling.ipynb [hunk 1]
```diff
@@ -0,0 +1,218 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "Styling"
```
Comment (Juanlu001) [inline docs/source/examples/styling.ipynb]: Could you use a level 1 heading here? Like `# Styling`, to be consistent with the other notebooks https://github.com/poliastro/poliastro/blame/5702ec937721389394a7721e862c10de654c4ad7/docs/source/examples/Catch%20that%20asteroid!.ipynb#L7

File: docs/source/examples/styling.ipynb [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
...JIh0DF0DPIKsnCxYyL1IRNCHkuDCWG6OPQByeST9DvGUJaARVDz+Dw/cP4+x9/R1ZpltBRCCHt1BDnIcgty8WN7BtCRyGk3aNi6BmcTj0Nr85esDO2EzoKIaSd6ufQD2sHrIWfpZ/QUQhp96gYaqKCigLczL6J/o79hY5CCGnHDCWGGOQ0CPo6+kJHIaTdo2Koic6lnYOKq9Dfob/QUQgh7VxuWS6+uvUVUgpThI5CSLtGxVATXUy/CEsDS8gt5UJHIYS0c5WqSmy8uRGnUk4JHYWQdo2KoSZaFrIM24Ztg4jRW0cIeb7sjO3gZuaGP9P+FDoKIe0a/UVvIh2RDpxMnYSOQQjpIHrb98bVrKsoVZQKHYWQdosWam2C3+79hnv597CgxwJqGSKapVwBMm8B2dHAw2SgNA+w9gVGb6jav/EF4GESIJIARhaAkRXgPhDov6hqf3YUYOYI6BkL9xqIVnnR/kVsj9yOK5lX0M+xn9BxCGmXqBhqgoMJB1FYWUiFEKlSlg/cPwsUpAG93qnadvTfQOplQNcEMHepKnZMa03BEPBKVYGkUgCluUBJDqCsqNqnVgNfDwQUZYBtAODaD3DrBziFALqGrf7yiHYIkgbBTM8M6SXpQkchpN1inHOhMzxXQUFB/OrVq82+ToWqAiE/hmCiz0Qs7LGwBZKRNqmiGIg5DETsBuJOAGoFYGAOzL8HiHWqWnb0TABTe6CpC/iqlEDsESDzNpD4J5Byuer6vecCg98H1CqAqwGx5Pm8NqK1VGoVxCJxi12PMXaNcx7UYhckpI2jlqGndCv7FirVlehp01PoKERIl74ETn5YVey88DbgMxKwD6wqhICqW2LPSqwD+I6s+g8AKkuA5AuAhWfV4/hTwO8zgMA3gOAZgIm0ea+FtBktWQgRQurTqvs9jLFhjLEYxlgcY2yRhv1zGWORjLHbjLETjLFWWyX1cuZliJgI3aXdW+spiTZIuQL8OBGIPlj1uOtrwBuHgdkRwNAPAacXnl9Lja4R4DEY6Pzon7mRBeDUCzi3GljrB+x9F8iLfz7PTbRKYkEiJh2YhIsZF4WOQki7pDXFEGNMDGAjgJcByAC8yhiT1TnsBoAgznkAgN0AVrZWPh2RDvra94WJrklrPSURUl48sOt14JvBQOoVoLygarupLeAcAogE+NGx6wZM+gF47xrQ7XXgzm
```
Comment (Juanlu001) [inline docs/source/examples/styling.ipynb]: We are getting a `No such kernel named python2` error:

https://circleci.com/gh/poliastro/poliastro/560

Perhaps rerun everything with Python 3? :wink:

Comment (Juanlu001) [inline docs/source/examples/styling.ipynb]: Remember that you can pass an `ax` to an `OrbitPlotter` object! http://docs.poliastro.space/en/latest/safe.html#poliastro.plotting.OrbitPlotter Perhaps you can rewrite this example using

```
fig, ax = plt.subplots(figsize=...)

plotter = plotting.OrbitPlotter(ax=ax)
...
```

?

File: docs/source/jupyter.rst [hunk 3]
```diff
@@ -16,3 +16,4 @@ Jupyter notebooks
    /examples/Using NEOS package.ipynb
    /examples/Visualizing the SpaceX Tesla Roadster trip to Mars.ipynb
    /examples/Natural and artificial perturbations.ipynb
+   /examples/styling.ipynb
```
Comment (Juanlu001) [inline docs/source/jupyter.rst]: Can we please rename this one to something more expressive, like `Customizing orbit plots.ipynb`? (Also, first letter in caps!)

**Comment by wselwood** [comment]
Could you let me know where you are seeing the problem with the kernel versions? I've run it on my local python 3 notebook server and nb viewer and I'm not seeing it. http://nbviewer.jupyter.org/github/poliastro/poliastro/blob/ff7db8d57c7e6a1c59baf28ecdf34f5de27e6e0f/docs/source/examples/Customising%20orbit%20plots.ipynb

**Comment by Juanlu001** [comment]
The issue of the kernel was fixed, previously I was seeing it here:

https://circleci.com/gh/poliastro/poliastro/560

Now all the stages passed, so I'm merging this! :tada:

**Pull request merged**